.eslintrc.js

```diff
@@ -80,7 +28,22 @@ module.exports = {
       files: 'packages/big-design-icons/**/*.tsx',
       rules: {
         'react/jsx-sort-props': 'warn',
+        '@typescript-eslint/no-unsafe-assignment': 'off',
       },
     },
   ],
+  settings: {
+    'import/resolver': {
+      typescript: {
+        alwaysTryTypes: true,
+        project: [
+          './packages/big-design/tsconfig.json',
+          './packages/big-design-icons/tsconfig.json',
+          './packages/big-design-theme/tsconfig.json',
+          './packages/docs/tsconfig.json',
+          './packages/examples/tsconfig.json',
+        ],
+      },
+    },
+  },
```

Comment:
Unable to resolve path to module '@test/utils' is thrown if removed. Thoughts?

Reply:
Ah... that's right I thought `@bigcommerce/eslint-config` doesn't support that 🤔  I guess this is fine, but could we update it to the following? (just a glob now instead of defining each one)
```
parserOptions: {
    project: 'packages/*/tsconfig.json',
    tsconfigRootDir: __dirname,
  },
  settings: {
    'import/resolver': {
      typescript: {
        alwaysTryTypes: true,
        project: 'packages/*/tsconfig.json',
      },
    },
  },
```